templates/agent/check_mk.service-drop-in.epp

```diff
@@ -0,0 +1,10 @@
+<%- |
```

Comment:
please add a first line like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
Fixed